.github/workflows/main.yml

```diff
@@ -54,7 +54,7 @@ jobs:
       run: vc-build Compile
 
     - name: Unit Tests
-      run: vc-build Test -TestsFilter "Category=Unit|Category=CI" -skip Restore+Compile
+      run: vc-build Test -TestsFilter "Category=Unit|Category=CI" -skip
```

Comment:
Do we need "-skip" param name if no values presented for it?

Reply:
Yes, without it, all targets on which Tests depend will run